### DISCUSSION TITLE
Missing coupled variables

### DISCUSSION THREAD

**Discussion by Krystalbbling** [idaholab/moose#26603] [Q&A Modules: Solid mechanics]
Hello dear experts，
Recently I tried to use MOOSE to simulate the thermal flow field, but encountered a warning.The details are as follows. I have not encountered this error before. The forces in the model only involve deformation caused by thermal expansion, but these modules do not exist in my input file. Maybe there are some problems when I configure or call them. , can you help me analyze it?
(![b28f0523c4f10b898385b2209a72422a](https://github.com/idaholab/moose/assets/109859157/85d72c3a-1d0f-48f3-b898-426c8b7c225a))
[fluidheat.txt](https://github.com/idaholab/moose/files/14008228/fluidheat.txt)

**Comment by liang-tech** [comment]
I neet this problem too when I upgrade the moose version to the newest.

**Comment by lindsayad** [comment]
Can you paste your input file instead of attaching it as a file?
